task/rpms-signature-scan/0.2/README.md

```diff
@@ -1,32 +1,33 @@
 # rpms-signature-scan task
 
-Scans RPMs in an image and provide information about RPMs signatures.
+## Description:
 
-## Parameters
```

Comment:
@avi-biton @amisstea do we still need this task in build-definitions or the one that we release from tekton-tools is enough.

The reason I'm asking is that I see there were some direct changes that were applied to the task in build-definitions, and I wonder if this discrepancy is something that we want to keep or we can deprecate this task.